src/Test.UnitTests.Sarif.Converters/FortifyUtilitiesTests.cs

```diff
@@ -11,7 +12,7 @@ public class FortifyUtilitiesTests
         public void FortifyUtilities_ParseFormattedContentText_Correct()
         {
             string content = @"<Content><Paragraph><b>General relativity</b> is the <pre>geometric theory of gravitation</pre> published by <i>Albert Einstein</i> in 1915 and the current description of <IfDef var=""ConditionalDescriptions""><ConditionalText attr=""value"">gravitation in modern physics</ConditionalText></IfDef>.<AltParagraph>General relativity generalizes special relativity and <Replace key=""Scientist.lastName""/>'s law of universal gravitation, providing a unified description of gravity as a geometric property of space and time, or spacetime. In particular, the curvature of spacetime is directly related to the energy and momentum of whatever matter and radiation are present.</AltParagraph>The relation is specified by the <code>Einstein field equations</code>, a system of <code>partial differential equations</code>.";
-            content += "\r\nAnd here's a garbage\n\r\n \n  \n\ndouble line break!</Paragraph></Content>";
+            content += $"{Environment.NewLine}And here's a garbage\n\r\n \n  \n\ndouble line break!</Paragraph></Content>";
```

Comment:
Do these also need to be replaced as well?

Reply:
When testing in WSL, the only part needed was the first one. Let me change and test again!

Reply:
just did one more changing replacing the other \r\n for the environment.newline